src/qt/receivecoinsdialog.cpp

```diff
@@ -44,20 +44,23 @@ ReceiveCoinsDialog::ReceiveCoinsDialog(const PlatformStyle *_platformStyle, QWid
 
     // context menu actions
     QAction *copyURIAction = new QAction(tr("Copy URI"), this);
+    QAction *copyAddressAction = new QAction(tr("Copy address"), this);
```

Comment:
style nit (see [clang-format-diff.py](https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy)):
```suggestion
    QAction* copyAddressAction = new QAction(tr("Copy address"), this);
```

Reply:
addressed in e348d7e